src/components/HorizontalTab/ChannelListTabItem.tsx

```diff
@@ -26,7 +27,8 @@ const ChannelListTabItem = (props: ChannelListTabItemProps) => {
     },
     name: {
       fontFamily: fonts.inter[500],
-      fontSize: 14
+      fontSize: 14,
+      color: props.type === 'SIGNED' ? colors.darkBlue : colors.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the name is now conditionally rendered based on the `type` prop. Ensure that the colors `colors.darkBlue` and `colors.black` are correctly defined and provide enough contrast for readability in all expected viewing conditions.
```diff
-      fontSize: 14
+      fontSize: 14,
+      color: props.type === 'SIGNED' ? colors.darkBlue : colors.black
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->